content/general/webhooks.textile

```diff
@@ -31,7 +31,7 @@ Ably currently supports three types of WebHooks:
 
 In order to prevent overloading your servers with requests, Ably batches WebHook data and issues a @POST@ request with @JSON@ (by default) data in the body of the request to your servers. See "binary support":#binary below for details on how to use @MsgPack@ for greater efficiency with a binary encoding. WebHook requests are typically published at most once per second per configured WebHook. See "WebHook transport details":#transport below for the specifics on throughput, restrictions and expected behavior of WebHooks.
 
-You can "configure WebHooks":#configure from the Reactor tab in your "account dashboard":https://support.ably.io/solution/articles/3000048664-how-do-i-access-my-account-dashboard on a per app basis which can apply to one or more channels in that app. WebHooks can be filtered by channel naming using a regular expression.
+You can "configure WebHooks":#configure from the "Reactor tab in your account dashboard":https://support.ably.io/solution/articles/3000074406-how-to-reach-the-reactor-page on a per app basis which can apply to one or more channels in that app. WebHooks can be filtered by channel naming using a regular expression.
```

Comment:
P.S. Why is the login image on https://support.ably.io/solution/articles/3000074406-how-to-reach-the-reactor-page stretched?